test/defaultbrowsercontext.spec.js

```diff
@@ -111,9 +111,9 @@ describe('launchPersistentContext()', function() {
       document.cookie = 'username=John Doe';
       return document.cookie;
     });
-    expect(documentCookie).toBe('username=John Doe');
     await page.waitForTimeout(2000);
```

Comment:
This is a textbook example of a test that is bound to be flaky. Would it make sense to have an explicit signal for blocked cookies?